app/src/main/java/org/thoughtcrime/securesms/conversation/v2/Util.kt

```diff
@@ -248,32 +246,6 @@ object Util {
         return result
     }
 
-    fun getSecretBytes(size: Int): ByteArray {
```

Comment:
Nice clean up!
Looking at the files there are a whole bunch more of seemingly unused functions. Might be worth removing those as well if they are truly unused.

Reply:
I removed `getRandomLengthBytes` because it creates a `SecureRandom` and it's probably better to delete an unused function than maintain it. I didn't want to get too distracted deleting all unused functions in files touched.